recipes/raxml/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: raxml
+  version: "8.2.4"
+build:
+  number: 0
+source:
+  fn: v8.2.4.tar.gz
+  url: https://github.com/stamatak/standard-RAxML/archive/v8.2.4.tar.gz
+requirements:
+  build:
+  run:
```

Comment:
Could you point me out to a recipe that you would consider to be a gold
standard on testing? I would try to do something similar. I have quite a
few tests for the galaxy tool version of this, so it will be trivial to do

On 9 December 2015 at 00:37, Johannes Köster <notifications@github.com>
wrote:

> In recipes/raxml/meta.yaml
> <https://github.com/bioconda/bioconda-recipes/pull/409#discussion_r47058712>
> :
>
> > @@ -0,0 +1,15 @@
> > +package:
> > +  name: raxml
> > +  version: "8.2.4"
> > +build:
> > +  number: 0
> > +source:
> > +  fn: v8.2.4.tar.gz
> > +  url: https://github.com/stamatak/standard-RAxML/archive/v8.2.4.tar.gz
> > +requirements:
> > +  build:
> > +  run:
>
> We try to always have a test case that checks whether the binary is
> working properly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/409/files#r47058712>.
>



-- 
"Democracy matters more than any currency arrangement." - Paul Krugman